src/content/en/fundamentals/accessibility/05-styling/styling-four.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: shared/narrow
+title: "Mobile Screen Readers"
+description: "Links to information about mobile-specific screen readers"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 4
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+There's not much to say about mobile-specific screen readers that hasn't been said about desktop screen readers. In this case, it might be best to let the experts speak for themselves. Have a look at the official Android and iOS accessibility docs for more information.
```

Comment:
Consider cutting this article or merging its content with the earlier one on VoiceOver/NVDA (semantics-navigation/index.markdown i believe)